.idea/.gitignore

```diff
@@ -18,6 +18,13 @@ workspace.xml
 
 # managed by gradle sync
 /gradle.xml
+/libraries
 
 # too much churn with teresology-modules each having their own repo
 /vcs.xml
+
+# generated by idea
+/jarRepositories.xml
+/uiDesigner.xml
+/modules.xml
+/modules
```

Comment:
ah, yes, not surprised we need to exclude `modules/`, I think I remember needing that in a different project.

this one so far I haven't seen anything show up in there except for `.iml` files, which are already ignored. other things end up in there too?

Reply:
i don't remember what there on my system.
See that:
https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore